src/Commands/GenerateSitemap.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace Portable\FilaCms\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Route;
+use Portable\FilaCms\Models\Form;
+use Portable\FilaCms\Facades\FilaCms;
+use Str;
+
+class GenerateSitemap extends Command
+{
+    protected $signature = 'fila-cms:generate-sitemap';
+
+    protected $description = 'Generate sitemap';
+
+    protected $exemptedWords = [
+        '_',
+        'storybook',
+        'horizon',
+        'livewire',
+        'two-factor-challenge',
+        'sanctum',
+        'filament',
+        'reset-password',
+        'media/{media}',
+        's/{slug}',
+        'link/{type}/{slug}',
+    ];
+
+    protected $exemptedMiddleware = [
+        'filament-actions',
+        'Illuminate\Session\Middleware\AuthenticateSession',
+        'Laravel\Pulse\Http\Middleware\Authorize',
+        'auth',
+        'auth:sanctum',
+        'auth:web'
+    ];
+
+    public function handle()
+    {
+        $routes = Route::getRoutes();
+
+        $filteredRoutes = [];
+        $this->generateSlugs('');
+
+        foreach ($routes as $key => $route) {
+            $shouldInclude = true;
+
+            // only GET routes
```

Comment:
Not a biggie, but a "neater" way to do this would be
`if(collect($route->methods())->search('GET') === false) continue`
And then the rest of your log isn't wrapped in the if